lib/exception.py

```diff
@@ -81,6 +81,10 @@ class RepositoryError(BaseException):
     # Subclass errors are in the form 0b0011xxx
     error_code = 24
 
+class FilesToValidateNotFound(BaseException):
+    DEFAULT_MESSAGE = (
+        "Files to be validated not found in base directory.")
+    error_code = 25
```

Comment:
Please use `error_code = 48` (0b00110000) and place this at the end of file, to keep consistency with other error binary codes. `25` is matching the prefix of a `RepositoryError`